engine/access/rpc/backend/backend_test.go

```diff
@@ -691,18 +762,31 @@ func (suite *Suite) TestGetAccount() {
 		Return(exeResp, nil).
 		Once()
 
+	ids := unittest.IdentityListFixture(1)
+	receipt := unittest.ReceiptForBlockFixture(&block)
+	suite.receipts.
+		On("ByBlockIDAllExecutionReceipts", blockID).
+		Return([]flow.ExecutionReceipt{*receipt}, nil).Once()
+	receipt.ExecutorID = ids[0].NodeID
```

Comment:
Same point here - modifying the `receipt` after we've copied it in the mocking call